Zend/zend_execute.c

```diff
@@ -1655,7 +1663,7 @@ static void zend_incdec_typed_ref(zend_reference *ref, zval *copy OPLINE_DC EXEC
 	}
 
 	if (UNEXPECTED(Z_TYPE_P(var_ptr) == IS_DOUBLE) && Z_TYPE_P(copy) == IS_LONG) {
-		zend_property_info *error_prop = zend_get_prop_not_accepting_double(ref);
+		zend_property_info *error_prop = error_prop = zend_get_prop_not_accepting_double(ref);
```

Comment:
I don't think you intended to do that :)

Reply:
😵